distro/kubernetes/apicurio-studio-ui-deployment.yaml

```diff
@@ -67,7 +67,7 @@ spec:
         - name: JAVA_TOOL_OPTIONS
           value: -Djava.net.preferIPv4Stack=true
         - name: APICURIO_UI_VALIDATION_CHANNELNAME_REGEXP
-            value: '{([^\x00-\x20\x7f"\'%<>\\^`{|}]|%[0-9A-Fa-f]{2}|\{[+#./;?&=,!@|]?((\w|%[0-9A-Fa-f]{2})(\.?(\w|%[0-9A-Fa-f]{2}))*(:[1-9]\d{0,3}|\*)?)(,((\w|%[0-9A-Fa-f]{2})(\.?(\w|%[0-9A-Fa-f]{2}))*(:[1-9]\d{0,3}|\*)?))*\})*'
```

Comment:
I got `error: error parsing apicurio-studio-ui-deployment.yaml: error converting YAML to JSON: yaml: line 71: found character that cannot start any token` because `'` was not escaped correctly. So I _think_ I have keept the regex pattern the same, but maybe you could check.